blocks/footer/footer.js

```diff
@@ -1,5 +1,20 @@
 import { readBlockConfig, decorateIcons } from '../../scripts/lib-franklin.js';
 
+/**
+ * Creates an anchor element with the given image as its child
+ * @param img
+ * @param {string} link
+ * @returns {HTMLAnchorElement} anchor
+ */
+
```

Comment:
```suggestion
```

Reply:
Thanks for fixing that Aravindo. I will watch for those next time